README.md

```diff
@@ -571,6 +574,29 @@ handle dependencies and CLDR loading manually yourself.
 
   Alias for `.relativeTimeFormatter( unit, options )( value )`.
 
+## Unit module
+
+-- **`.unitFormatter( unit, options )`**
```

Comment:
The two dashes at the start of the line should be a single dash, to match the formatting from other API entries.